source/transactions-convenient-api/tests/callback-aborts.yml

```diff
@@ -89,6 +89,8 @@ tests:
     # Session state will be NO_TXN when callback returns to withTransaction
     description: withTransaction still succeeds if callback aborts and runs extra op
     useMultipleMongoses: true
+    clientOptions:
+      retryWrites: false
```

Comment:
Is this necessary? The APM assertion below intentionally avoids asserting a `txnNumber` field in the final insert after the abort:

> This test is agnostic about retryWrites, so we do not assert the txnNumber. If retryWrites=true, the txnNumber will be incremented from the value used in the previous transaction; otherwise, the field will not be present at all.

Reply:
Please add a comment explaining why this is necessary and remove the "This test is agnostic about retryWrites" comment further down in this file.

Likewise for `callback-commits.yml`.